hack/install-database.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  selector:
+    matchLabels:
+      app: postgresql
+  template:
+    metadata:
+      labels:
+        app: postgresql
+    spec:
+      containers:
+        - name: postgresql
+          image: postgres:16
+          ports:
+            - containerPort: 5432
+          env:
+            - name: POSTGRES_DB
+              value: root
+            - name: POSTGRES_USER
+              value: root
+            - name: POSTGRES_PASSWORD
+              value: password  # notsecret
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+  selector:
+    app: postgresql
+
+EOF
+
+kubectl rollout status deployment --namespace=databases --timeout=30s
+sleep 15  # Wait for PostgreSQL to be really ready
+
+echo "CREATE USER kubearchive WITH ENCRYPTED PASSWORD 'P0stgr3sdbP@ssword';" | kubectl exec -i -n databases $(kubectl get -n databases pod -o name) -- psql -h localhost
+echo "CREATE DATABASE kubearchive WITH OWNER kubearchive;" | kubectl exec -i -n databases $(kubectl get -n databases pod -o name) -- psql -h localhost
+cat database/ddl.sql | kubectl exec -i -n databases $(kubectl get -n databases pod -o name) -- psql -h localhost --user=kubearchive kubearchive
+
+kubectl port-forward -n databases svc/postgresql 5432:5432 &
```

Comment:
In the previous `kubectl` commands you calculate the name, here it is hard coded as "svc/postgresql". Why?

Reply:
Because the other `kubectl` commands were pointing to a Pod deployed by a Deployment. Those pods get numbers with a short uid (for example `postgresql-57d5dd655f-bxmpz`) so I need to retrieve the pod name dynamically. However I will use a label to filter which pods can be returned as it will better indicate the intention.

In this `kubectl` command I'm pointing to a service that has a fixed name.